### DISCUSSION TITLE
Locally ignored files are not ignored

### DISCUSSION THREAD

**Discussion by jirih** [gitextensions/gitextensions#8911] [Q&A]
1. open commit dialog
2. right click on a file in the left top panel (Working directory changes)
3. choose Add file to .git/info/exclude

At this point you can check the file is really listed in .git/info/exclude.

However the file is still visible in the list of files like it is not ignored.

**Comment by pmiossec** [comment]
I'm pretty sure that the file is already tracked by git.
If that is the case, that is normal and you don't understand well how ignore files works with git.

A already tracked file can't be ignored.

You could share screenshots to be sure...

**Comment by mstv** [comment]
Perhaps "skip-worktree" matches your use case better.
